cpp/include/cudf/copying.hpp

```diff
@@ -627,10 +627,25 @@ std::unique_ptr<table> boolean_mask_scatter(
  *
  * @returns Returns a table by scattering `input` into `target` as per `boolean_mask`.
  */
- std::unique_ptr<table> boolean_mask_scatter(
+std::unique_ptr<table> boolean_mask_scatter(
     std::vector<std::reference_wrapper<scalar>> const& input, table_view const& target,
     column_view const& boolean_mask,
     rmm::mr::device_memory_resource *mr = rmm::mr::get_default_resource());
 
+/**
+ * @brief Get the element at specified index from a column
+ * 
+ * @throws cudf::logic_error if `index` is not within the range`[0, input.size())`
+ * 
+ * @param input Column view to get the element from
+ * @param index Index into `input` to get the element at
+ * @param mr Optional, The resource to use for all returned allocations
+ * @return std::unique_ptr<scalar> Scalar containing the single value
+ */
+std::unique_ptr<scalar> get_element(
```

Comment:
Why is this not just a method of `column_view`?  Seems to me we should just add `element()` to `column_view` just like we have in `column_device_view`,  except that it would copy to the scalar. @jrhemstad opinion?

At a minimum, I don't see why this function is in `copying.hpp`. Should be in something related to column views.